nuxeo-runtime/nuxeo-stream/src/main/java/org/nuxeo/lib/stream/log/LogTailer.java

```diff
@@ -82,6 +82,16 @@
      */
     void seek(LogOffset offset);
 
+    /**
+     * Look up the offset for the given partition by timestamp.
+     * The position is the earliest offset whose timestamp is greater than or equal to the given timestamp.<p/>
+     * The timestamp used depends on the implementation, for Kafka this is the LogAppendTime.
+     * Returns null if no record offset is found with appropriate timestamp.
+     *
```

Comment:
This important description need to be duplicated in the command help description so admin don't have to read the java doc.